docs/build_docs/source/errors/index.rst

```diff
@@ -51,9 +51,50 @@ in your input file have correct values. Remember that ligands needs to have a un
 Connections Error
```

Comment:
I really like the section you have added. It is going to be very helpful for users. Good work!